client/extensions/zoninator/components/forms/zone-details-form/index.jsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { reduxForm } from 'redux-form';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormButton from 'components/forms/form-button';
+import FormLabel from 'components/forms/form-label';
+import ReduxFormTextarea from 'components/redux-forms/redux-form-textarea';
+import ReduxFormTextInput from 'components/redux-forms/redux-form-text-input';
+import SectionHeader from 'components/section-header';
+
+const form = 'extensions.zoninator.zoneDetails';
+
+const ZoneDetailsForm = ( {
+	handleSubmit,
+	label,
+	onSubmit,
+	submitting,
+} ) => {
+	const save = data => onSubmit( form, data );
+
+	return (
+		<form onSubmit={ handleSubmit( save ) }>
+			<SectionHeader label={ label }>
+				<FormButton
+					compact
+					disabled={ submitting }
+					isSubmitting={ submitting }>
+					{ translate( 'Save' ) }
+				</FormButton>
+			</SectionHeader>
+			<CompactCard>
+				<FormFieldset>
+					<FormLabel htmlFor="name">{ translate( 'Zone name' ) }</FormLabel>
+					<ReduxFormTextInput name="name" />
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLabel htmlFor="description">{ translate( 'Zone description' ) }</FormLabel>
+					<ReduxFormTextarea name="description" />
+				</FormFieldset>
+			</CompactCard>
+		</form>
+	);
+};
+
+ZoneDetailsForm.propTypes = {
+	label: PropTypes.string.isRequired,
+	onSubmit: PropTypes.func.isRequired,
+};
+
+const createReduxForm = reduxForm( {
+	form,
+	validate: ( data ) => {
+		const errors = {};
+
+		if ( ! data.name ) {
+			errors.name = translate( 'Zone name cannot be empty.' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Product name cannot be empty.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).